onnx/defs/generator/defs.cc

```diff
@@ -119,6 +119,81 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
+static const char* EyeLike_ver9_doc = R"DOC(
+Generate a 2D tensor (matrix) with ones on the diagonal and zeros everywhere else. Only 2D 
+tensors are supported, i.e. input T1 must be of rank 2.  The shape of the output tensor is the 
+same as the input tensor. The data type can be specified by the 'dtype' argument. If 
+'dtype' is not specified, then the type of input tensor is used.
+
+The 'dtype' argument must be one of the data types specified in the 'DataType' enum field in the
+TensorProto message and be valid as an output type.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    EyeLike,
+    9,
+    OpSchema()
+        .SetDoc(EyeLike_ver9_doc)
+        .Attr(
```

Comment:
Do you also want to add `k` parameter to indicate the index of the diagonal: 
0 (the default) refers to the main diagonal, a positive value refers to an upper diagonal, and a negative value to a lower diagonal.

Reply:
Sure. I will add that.

Reply:
Added.